spec/factories/globus_destination.rb

```diff
@@ -14,7 +14,7 @@
     end
 
     trait :stale do
-      created_at { 1.month.ago }
+      created_at { 2.months.ago }
       deleted_at { nil }
```

Comment:
since some types of stale destinations can now be 1 month old, let's make sure they are extra extra stale